session_3/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Articles</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Tags</th>
+      <th>Topic</th>
+      <th>Content</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @articles.each do |article| %>
+      <tr>
+        <td><%= article.title %></td>
+        <td><%= article.tags %></td>
+        <td><%= article.topic %></td>
+        <td><%= article.content %></td>
+        <td><%= link_to 'Show', article %></td>
+        <%if can? :update, article, user_id: current_user.id || current_user.admin%>
```

Comment:
Instead of checking if the current user is admin in view, give the admin user the ability to modify all articles in the abilities file.

Reply:
I'll change it